hbp_knowledge/antalarmin/dinopoulou2013.bel

```diff
@@ -73,32 +73,53 @@ DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifact
 SET Citation = {"PubMed", "23211705"}
 
 SET Evidence ="At all embryo stages, significant differences were noted between the CRH 10􏰁7 and control groups as well as be- tween the CRH 10􏰁7 and CRH 10􏰁7/antalarmin 10􏰁6 groups, with the rates being lower in the former. "
+SET Confidence = "High"
 a(MESH:antalarmin) -> a(MESH:"Embryonic Structures")
+UNSET Confidence
 
 SET Evidence ="With regard to the morula/blastocyst stage, the rates in the CRH 10􏰁7 group were lower compared with the control (P 􏰄 0.003) and lower compared with the CRH 10􏰁7/antalarmin 10􏰁6 group (P 􏰃 0.001) (Table 2)."
+#control : preantral follicle
+#Group II : preantral follicle with Crh
+#Group III:  preantral follicle with Crh/antalarmin
+SET Confidence = "High"
 a(MESH:antalarmin) -> a(MESH:Morula)
 a(MESH:antalarmin) -> a(MESH:Blastocyst)
+UNSET Confidence
 
 SET Evidence ="Interestingly, the addition of a 10-fold excess of the CRH-R1 antagonist, antalarmin, overcame the negative effect of CRH on estradiol release (d 5, P 􏰄 0.009; d 7, P 􏰄 0.010; d 9, P 􏰃 0.001; and d 11, P 􏰄 0.002), indicating that the suppressive effect of CRH was receptor mediated (Fig. 1)."
-p(MGI:Crh) -| a(MESH:Estradiol)
+SET Confidence = "Medium"
+p(MGI:Crh) -| sec(a(MESH:Estradiol))
 a(MESH:antalarmin) -| act(p(MGI:Crh))
+UNSET Confidence
 
+#Uncertain of use of CGB3 or HCG for the naming convention of beta-hCG
 SET Evidence ="The addition of the CRH-R1 an- tagonist antalarmin in a 10-fold higher concentration compared with that of CRH overcame the effect of CRH on beta􏰇-hCG release (d 11 beta􏰇-hCG 185.6 􏰅 6.087 mIU/ml, P 􏰃 0.001) indicating that the suppressive effect of CRH was recep- tor mediated (Fig. 3)."
+SET Confidence = "High"
 p(MGI:Crh) reg p(HGNC:CGB3)
 a(MESH:antalarmin) -| act(p(MGI:Crh))
```

Comment:
I would maybe add a statement that involves the fact that the effect is receptor mediated:
act(p(MGI:Crhr1)) -| p(HGNC:CGB3)